donation-api/src/main/java/org/interfaithsanctuary/donationapi/AlertJPA.java

```diff
@@ -0,0 +1,19 @@
+package org.interfaithsanctuary.donationapi;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Alert {
+    @Id
+    @GeneratedValue(strategy=GenerationType.AUTO)
+    private int alert_id;
+    private String alert_title;
+    private String alert_body;
+    private int callout_id;
+    private int need_id;
+    private Date create_date;
+    private Date last_pushed_date;
```

Comment:
I would create some getter / setter methods for the variables. Also be sure to annotate the variables with the @Column annotation to specify their proper database column for clarity.